src/ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -5,8 +5,8 @@
     <RootNamespace>Neo.Plugins</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\RpcServer\RpcServer.csproj" AdditionalProperties="IncludeSettingsFileOutput=False"/>
```

Comment:
Anyway, we have to revert this. Otherwise config.json for applicationlog will have conflict with the one for RpcServer since they will build in the same path.